PaddleCV/PaddleDetection/ppdet/core/workspace.py

```diff
@@ -125,7 +125,7 @@ def register(cls):
     return cls
 
 
-def create(cls_or_name, **kwargs):
+def create(cls_or_name, prefix_name='', **kwargs):
```

Comment:
better to put weight_prefix_name in the YAML config, then pass the args by __init__, not this `create` interface?

Reply:
Fixed.